pkg/system/phase4_configuring.go

```diff
@@ -455,6 +455,37 @@ func (r *Reconciler) setDesiredEndpointMounts(podSpec *corev1.PodSpec, container
 	return nil
 }
 
+// Duplicate code from validation.go namespacetore pkg.
+// Cannot import the namespacestore pkg, because the pkg imports the system pkg
+// TODO remove the code
+func (r *Reconciler) validateNsStoreNSFS(nsStore *nbv1.NamespaceStore) bool {
+	nsfs := nsStore.Spec.NSFS
+
+	if nsfs == nil {
```

Comment:
I think this check is repeating itself - but as we want to remove this code later - I think it's fine

Reply:
Could you explain me pls, why do you think it's repeating?

Reply:
OK, We will have the status in the namespacestore describe. leave it